packages/pretty-format/src/index.js

```diff
@@ -389,7 +389,9 @@ const getColorsHighlight = (options: OptionsReceived): Colors =>
       colors[key] = color;
     } else {
       throw new Error(
-        `pretty-format: Option "theme" has a key "${key}" whose value "${value}" is undefined in ansi-styles.`,
```

Comment:
It was easier to read before, IMO

Reply:
Yeah but looks like it exceeds printWidth

Reply:
It does, but it's way easier to read.

Same as long string are unbroken, I would prefer there to not be a break in long template strings, at least as long as there's just single variables in the `${}` parts